app/src/main/res/values/colors.xml

```diff
@@ -79,4 +79,7 @@
     <color name="primary_dark">#313131</color>
     <color name="accent">#3797C7</color>
 
+    <color name="appcompat_secondary_text_default_material_light">#8a000000</color>
+    <color name="appcompat_secondary_text_disabled_material_light">#24000000</color>
+
```

Comment:
No, these 2 color resources are declared as private in appcompat.
So intelliJ (and i think lint too) suggest to not use them directly as they may disappear in future releases.